### PR TITLE
61188: Added info to hot spares section on how to add spare devices.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/storage.rst [hunk 1]
```diff
@@ -1803,6 +1803,14 @@ Hot spares can be added to a volume during or after creation. On
 %brand%, hot spare actions are implemented by
 `zfsd(8) <https://www.freebsd.org/cgi/man.cgi?query=zfsd>`__.
 
+Add a spare by going to
+:menuselection:`Storage --> Volume Manager`.
+Select the volume to extend from the :guilabel:`Volume to extend` drop
+down. Choose a disk from the list of :guilabel:`Available disks` and
```
Comment (wblock) [inline userguide/storage.rst]: Check the docteam guide, I think we use "drop-down", but verify.

File: userguide/storage.rst [hunk 2]
```diff
@@ -1803,6 +1803,14 @@ Hot spares can be added to a volume during or after creation. On
...
+click **+** to add that disk to the volume. Select :literal:`spare` in
```
Comment (wblock) [inline userguide/storage.rst]: GUI element should be marked up that way, and contents of fields are in single stars:
```suggestion
click :guilabel:`+` to add that disk to the volume. Select *spare* in
```

File: userguide/storage.rst [hunk 3]
```diff
@@ -1803,6 +1803,14 @@ Hot spares can be added to a volume during or after creation. On
...
+the :guilabel:`Volume layout` drop down. Click :guilabel:`Extend Volume`
+to finalize the process of adding a hot spare to the selected volume.
```
Comment (wblock) [inline userguide/storage.rst]: Wordy...
```suggestion
to the hot spare.
```

File: userguide/storage.rst [hunk 4]
```diff
@@ -1803,6 +1803,14 @@ Hot spares can be added to a volume during or after creation. On
 %brand%, hot spare actions are implemented by
 `zfsd(8) <https://www.freebsd.org/cgi/man.cgi?query=zfsd>`__.
 
+Add a spare by going to
+:menuselection:`Storage --> Volume Manager`.
+Select the volume to extend from the :guilabel:`Volume to extend` drop
+down. Choose a disk from the list of :guilabel:`Available disks` and
+click :guilabel:`+` to add that disk to the volume. Select *spare* in
+the :guilabel:`Volume layout` drop down. Click :guilabel:`Extend Volume`
+to finalize the process of adding a hot spare to the selected volume.
```
Comment (a-stjohn) [inline userguide/storage.rst]:
```suggestion
to add the hot spare to the selected volume.
```

File: userguide/storage.rst [hunk 5]
```diff
@@ -1803,6 +1803,14 @@ Hot spares can be added to a volume during or after creation. On
 %brand%, hot spare actions are implemented by
 `zfsd(8) <https://www.freebsd.org/cgi/man.cgi?query=zfsd>`__.
 
+Add a spare by going to
+:menuselection:`Storage --> Volume Manager`.
+Select the volume to extend from the :guilabel:`Volume to extend` drop
```
Comment (a-stjohn) [inline userguide/storage.rst]:
```suggestion
Select the volume to extend from the :guilabel:`Volume to extend` drop-
```

File: userguide/storage.rst [hunk 6]
```diff
@@ -1803,6 +1803,14 @@ Hot spares can be added to a volume during or after creation. On
 %brand%, hot spare actions are implemented by
 `zfsd(8) <https://www.freebsd.org/cgi/man.cgi?query=zfsd>`__.
 
+Add a spare by going to
+:menuselection:`Storage --> Volume Manager`.
+Select the volume to extend from the :guilabel:`Volume to extend` drop-
+down. Choose a disk from the list of :guilabel:`Available disks` and
+click :guilabel:`+` to add that disk to the volume. Select *spare* in
+the :guilabel:`Volume layout` drop down. Click :guilabel:`Extend Volume`
```
Comment (a-stjohn) [inline userguide/storage.rst]:
```suggestion
the :guilabel:`Volume layout` drop-down. Click :guilabel:`Extend Volume`
```

File: userguide/storage.rst (hunk 4 above)
Comment (wblock) [reply userguide/storage.rst]: Oops, was missing a word, meant to say "to add the hot spare."  I think context makes it unnecessary to repeat "to the selected volume".

File: userguide/storage.rst [hunk 7]
```diff
@@ -1803,6 +1803,14 @@ Hot spares can be added to a volume during or after creation. On
...
+to add the hot spare to the selected volume.
```
Comment (a-stjohn) [inline userguide/storage.rst]:
```suggestion
to add the hot spare.
```

**Pull request merged**